qcengine/programs/torchani.py

```diff
@@ -172,7 +173,7 @@ def compute(self, input_data: "AtomicInput", config: "TaskConfig") -> "AtomicRes
         ret_data["extras"] = input_data.extras.copy()
         ret_data["extras"].update(
             {
-                "ensemble_energies": energy_array.detach().numpy(),
+                "ensemble_energies": energy_array.cpu().detach().numpy(),
                 "ensemble_energy_avg": energy.item(),
```

Comment:
To make sure I understand correctly, calling `item` will move the "energy" tensor to the cpu? You don't need a `.cpu()` statement, right?

Reply:
Yes, calling `item()` would move the tensor to CPU, you are absolutely right.